xml/System.IO.Compression/ZipFileExtensions.xml

```diff
@@ -178,7 +179,7 @@
 ## Remarks  
  The new entry in the archive contains the contents of the file specified by `sourceFileName`. If an entry with the specified name (`entryName`) already exists in the archive, a second entry is created with an identical name. The <xref:System.IO.Compression.ZipArchiveEntry.LastWriteTime%2A> property of the entry is set to the last time the file on the file system was changed.  
   
-   
+ When `ZipArchiveMode.Update` is present, the size limit of an entry is limited to <xref:System.Int32.MaxValue?displayProperty=nameWithType>. This limit is because update mode uses a <xref:System.IO.MemoryStream> internally to allow the seeking required when updating an archive, and <xref:System.IO.MemoryStream> has a maximum equal to the size of an int.  
```

Comment:
not related to this PR but a general comment for us to discuss. When the remarks are the same for overloads, should we just move it to the remarks of the overload itself instead of repeating for each one? /cc @rpetrusha